src/Discord.Net.Commands/Module.cs

```diff
@@ -16,6 +19,7 @@ public class Module
         public string Remarks { get; }
         public IEnumerable<Command> Commands { get; }
         internal object Instance { get; }
+        public ConcurrentDictionary<Type, TypeReader> OverridingTypeReaders { get; }
```

Comment:
No need for this to be concurrent, it should be immutable after the module has been created